examples/fields.go

```diff
@@ -3,14 +3,14 @@ package main
 
 import (
 	"fmt"
-
+	"github.com/hpe-storage/nimble-golang-sdk/pkg/client"
```

Comment:
The need to add this import is a clue that we've defined the functions in the wrong place.  :)